doc/source/reference/routines.linalg.rst

```diff
@@ -18,6 +18,25 @@ or specify the processor architecture.
 .. _OpenBLAS: https://www.openblas.net/
 .. _threadpoolctl: https://github.com/joblib/threadpoolctl
 
+Frequently asked question about `numpy.linalg` include "Why do both NumPy and
+SciPy have `linalg` submodules, with duplicated functions?" and "Which should
+I use?"  The following is from the SciPy FAQ:
+
+* `scipy.linalg` is a more complete wrapping of Fortran LAPACK using f2py.
+* One of the design goals of NumPy was to make it buildable without a Fortran
+  compiler, and if you don’t have LAPACK available, NumPy will use its own
+  implementation. SciPy requires a Fortran compiler to be built, and heavily
+  depends on wrapped Fortran code.
+* The ``linalg`` modules in NumPy and SciPy have some common functions but
+  with different docstrings, and `scipy.linalg` contains functions not found
```

Comment:
"Different docstrings" but the same interfaces, or are the interfaces different as well?

Reply:
I suspect my work on vectorization over stacks of matrices didn't get ported to scipy

Reply:
There's some other differences that need to be resolved, e.g. the `qr` keywords

Reply:
Docstrings are less relevant I'd say

Reply:
I pushed an update with a shorter comparison of numpy.linalg and scipy.linalg.